.gitignore

```diff
@@ -7,3 +7,4 @@ _vimrc_local.vim
 config/config.php
 test/coverage
 .vagrant
+composer.lock
```

Comment:
All dependencies which are used by the application have fixed version numbers in `composer.json`. Don`t get the advantage of addtionally adding composer.lock.